### PR TITLE
Accurev Repository Fixes

### PR DESCRIPTION
**Pull request by ozankiraz**
Accurev Repository Fixes, minor ui fix

### CODE REVIEW

**Comment by ozankiraz** [comment]
#1356 issue was still valid even though it was closed, Inline CSS Style unit changed to em,
Accurev repository regex pattern fixed, absolute paths are used to run accurev command line now, so annotations can be read properly

16523616_1816726785251849_2034392541_o


After fix : 
annotation css fix

**Comment by ozankiraz** [comment]
Note to reviewer : I'm not aware of any issues with Accurev repository, so I cannot mention any issue number,

I had a issue when Indexer are run with -H parameter, working directory is top directory containing all Accurev work spaces, but this directory is not part of any work space, so history and annotations cannot be read, I fixed it by passing working directory as file's parent directory,

**Comment by vladak** [comment]
Thanks for the stuff. Before this can be merged in we need your OCA (Oracle Contributor Agreement).

**Comment by ozankiraz** [comment]
Hi Vladak,
I'll send OCA and get back to you when process is completed.
Regards,
Ozan

**Comment by vladak** [comment]
Okay, just let us know once you sent the document out and I will accept. In the meantime maybe @tulinkry can look at the UI changes ?

**Comment by ozankiraz** [comment]
@vladak I just sent OCA, let me know about how to progress
@tulinkry I'm not good at CSS stuff, let me know if fix is not ideal, screenshot are provided above.

**Comment by vladak** [comment]
Once @tulinkry has his say w.r.t. the style I will merge this in.

**Comment by tulinkry** [comment]
On my screen there is too big margin for it. For me the `ex` works fine. When I change the font size sometimes I get out of the box. Roughly the value of `1ex` should correspond to something like `0.5em`. So actually using `em` instead of `ex` is a little bit more over. Try to play with it more...I tried values around 0.7 and it looked quite ok.

Or maybe use 1.2 of `ex` and try to see if that's better. The letter "x" is not the best representative of an alphabet in this case.

TLDR: I'm not against the change to `em` but this has too big margin.

screenshot from 2017-02-15 12-54-57

**Comment by ozankiraz** [comment]
I'll try with numberOfCharacters * 1.2 ex and I'll update the screenshot when I get back to home. I agree that margin big. I'll keep you updated

**Comment by ozankiraz** [comment]
@tulinkry 
After some trial and error I decided to go with number of character * 2 + 1 ex

It looks like this:

og_ui_annotation_columns_fix

Let me know about suggestions

**Comment by tulinkry** [comment]
I don't know what kind of settings you use but this looks like this screenshot

screenshot from 2017-02-16 14-36-24

**Comment by ozankiraz** [comment]
@tulinkry This is weird :/ In virtual machine I'm using it looks exactly like I posted.
It's win 2008 VM, Firefox 51 Clean install. I also tried it on Chrome.
I'll roll back it to original one since my fix was mainly about Accurev Repository, so my pull request can be merged. I'll take care of ui thing later

**Comment by ozankiraz** [comment]
@tulinkry My bad, I deployed wrong war file and got some weird results.

I tried it with number of characters * 0.7 em it works great,
Updating pull request

**Comment by vladak** [comment]
merged.

Pls update the issue with your OCA once you get it. Thanks !

**Comment by tulinkry** [comment]
Ok. Thank you
